docs/user/reference/yaml/metrics.md

```diff
@@ -169,6 +172,13 @@ May be one of the following values:
 
 ### Optional parameters
 
+#### `tags`
```

Comment:
This is not a top-level parameter, but under the `metadata` parameter. To avoid confusion we should call this out here.